addon/templates/components/frost-multi-select.hbs

```diff
@@ -17,7 +17,7 @@
     </div>
     <div class='multi-status'>
       <span class='number-selected'>{{selected.length}} selected</span>
-      <a href='#' class='clear' tabIndex={-1} onClick={{action "clearSelection"}}>Clear</a>
+      <a class='clear' tabIndex={-1} onClick={{action "clearSelection"}}>Clear</a>
```

Comment:
@cstolli Since this was probably you, any objection to removing the `href` altogether? I've never really used `href="#"`, but I know there at least *was* a legit reason to, not sure if it's still applicable.